agrirouter-api-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs

```diff
@@ -125,17 +121,18 @@ public IEnumerable<string> ChunkMessageContent(string base64MessageContent, int
             var chunks = new List<string>();
             var source = Encoding.UTF8.GetBytes(base64MessageContent);
             byte[] chunk;
-            for(var i = 0; i < source.Length; i+=chunkSize)
+            for (var i = 0; i < source.Length; i += chunkSize)
             {
                 if (i + chunkSize > source.Length) continue;
```

Comment:
I'ld prefer 

`if(i+ chunkSize > source.Length) {
      continue;
}`

or even 

`if( i + chunkSize < = source.Length){
                   chunk = new byte[chunkSize];
                Array.Copy(source, i, chunk, 0, chunkSize);
                chunks.Add(Encoding.UTF8.GetString(chunk)); 
}
`

Those one-liner-Statements without {} lead to issues in code maintenance quite often

Reply:
Should mark the new code, no idea why it's on the old code.

Reply:
This is a common pattern in .NET - therefore no fix applied.